<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QLabel" name="path_file">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="prev_img">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="result_img">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>choose_file_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>process_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>choose_file_clicked()</slot>
  <slot>process_clicked()</slot>
 </slots>
</ui>
